deployment/ccip/changeset/testhelpers/cciptesthelpertypes/types.go

```diff
@@ -0,0 +1,218 @@
+package cciptesthelpertypes
+
+import (
+	"fmt"
+	"maps"
+	"math/rand"
+	"slices"
+	"sort"
+
+	cciptypes "github.com/smartcontractkit/chainlink-ccip/pkg/types/ccipocr3"
+	"golang.org/x/exp/constraints"
+)
+
+// MinRoleDONSize represents the size of the smallest possible CCIP role DON.
+const MinRoleDONSize = 4
+
+// NChain returns the number of chains that a given fChain value will support.
```

Comment:
Is this more accurate?
```suggestion
// NChain returns the number of readers for a chain required based on a given fChain value.
```